coral-component-tablist/src/styles/index.styl

```diff
@@ -27,6 +27,13 @@
   ._coral-Tabs-itemLabel:not(:empty) {
     display: inline;
   }
+
+  ._coral-Tabs-itemInvalidIcon {
+    display: inline;
+    margin-right: 0;
+    margin-left: 5px;
```

Comment:
can you use a pre-defined css var instead and check that it works in large scale mode ?